src/api/java/appeng/api/implementations/items/IMemoryCard.java

```diff
@@ -41,11 +43,16 @@
 	 * Configures the data stored on the memory card, the SettingsName, will be
 	 * localized when displayed.
 	 *
+	 * The data can contain an optional string with the key "tooltip", which will be used as
+	 * unlocalized string to display it after the settings name.
+	 *
+	 * The data can contain an optional intArray using "colorCode" to be displayed on the model itself.
+	 * It needs to have exactly 8 elements representing the ordinal of the matching {@link AEColor}.
+	 * The first 4 values represent the bottom row, left to right. The second 4 the top row.
+	 *
 	 * @param is item
 	 * @param SettingsName unlocalized string that represents the tile entity.
-	 * @param data may contain a String called "tooltip" which is is a
-	 * unlocalized string displayed after the settings name, optional
-	 * but can be used to add details to the card for later.
+	 * @param data the NBT tag, refer to the normal comment for special keys.
 	 */
 	void setMemoryCardContents( ItemStack is, String SettingsName, NBTTagCompound data );
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS00117)